README.md

```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
 ## Why Superset
 
 Superset provides:
-* An intuitive interface to explore and visualize datasets, and
-    create interactive dashboards.
-* A wide array of beautiful visualizations to showcase your data.
-* Easy, code-free, user flows to drill down and slice and dice the data
-    underlying exposed dashboards. The dashboards and charts act as a starting
-    point for deeper analysis.
-* A state of the art SQL editor/IDE exposing a rich metadata browser, and
-    an easy workflow to create visualizations out of any result set.
-* An extensible, high granularity security model allowing intricate rules
-    on who can access which product features and datasets.
-    Integration with major
-    authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
-* A lightweight semantic layer, allowing to control how data sources are
-    exposed to the user by defining dimensions and metrics
-* Out of the box support for most SQL-speaking databases
-* Deep integration with Druid allows for Superset to stay blazing fast while
-    slicing and dicing large, realtime datasets
-* Fast loading dashboards with configurable caching
+
+Viewed
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern, 
+enterprise-ready business intelligence web application. It is fast, lightweight, intuitive, 
+and loaded with options that make it easy for users of all skill sets to explore and visualize 
+their data, from simple pie charts to highly detailed deck.gl geospatial charts.
+
+[**Why Superset**](#why-superset) |	[**Why Superset**](#why-superset) |
+[**Supported Databases**](#supported-databases) |	[**Supported Databases**](#supported-databases) |
+
+## Why Superset	## Why Superset
+
+Superset provides:	Superset provides:
```

Comment:
odd doubles, wondering what happened here. I highly recommend doing a first sanity pass on your PR when opening the PR on the step prior to "Create PR"